src/common/keyboard.js

```diff
@@ -92,7 +92,7 @@ if (!('key' in window.KeyboardEvent.prototype)) {
   ( window.KeyboardEvent.prototype
   , 'key'
   , { get: getKey
-    /*::, value: void(0)*/
+    , value: void(0)
```

Comment:
This is just silly: flow wants the object literal to have a `value` field for no good reason whatsoever. I re-added the comment, with an explanation.